bot/handlers.py

```diff
@@ -178,3 +355,20 @@ async def get_chat_state(message: Message, chat_state: ChatState):
             ),
             parse_mode=ParseMode.HTML,
         )
+
+
+def handle_user_responses(
+    scheduler: AsyncIOScheduler, send_message: SendMessage, router: Router
+):
+    @router.message(
+        HasMessageUserUsername(), HasChatState(), IsReplyToMeetingMessage()
+    )
+    async def set_meetings_time(
+            message: Message, username: str, chat_state: ChatState, replied_meeting_msg_num: int
+    ):
+        if message.from_user.username in chat_state.users:
+            user = await get_user(chat_state, username)
+
+            reply_msg_key = f"has_replied_to_msg_{replied_meeting_msg_num}"
```

Comment:
`reply_msg_key` won't be renamed when you rename `ChatUser` fields. You should use `match .. case` on `replied_meeting_msg_num`.

Reply:
As I understood you want me to rewrite it like this:
```
match replied_meeting_msg_num:
            case 1:
                user.has_replied_to_msg_1 = True
            case 2:
                user.has_replied_to_msg_2 = True
            case 3:
                user.has_replied_to_msg_3 = True
```
I don't really get how it could help us with renaming fields in `ChatUser`. Let's say we rename filed `has_replied_to_msg_N` to just `replied_to_msg_N`. Code variant with match case will not work after renaming, similarly to the code with f-string. I still have to rename those fields, because they are statically written in the code. The only thing here: I have to rename 3 lines of code in match case variant and 1 line of code with f-string variant.
Or I just stupid and don't get how to do it properly with match case, to prevent code editing after renaming fields.

Reply:
> `reply_msg_key` won't be renamed when you rename ChatUser fields.

Here, I meant the value won't be changed automatically.

> Code variant with match case will not work after renaming, similarly to the code with f-string.

There will be a type error if you try to assign a value to a non-existent `user` field.

Reply:
Actually, I think it's better to use a `set` instead of individual fields.